### PR TITLE
#160011173 setup sass pre-processing

### PR DESCRIPTION
**Pull request by migot01**
#### What does this PR do?
Setup sass pre-processing

#### Description of tasks to be completed
-Install sass loader and dependancies
-Add sass loader to webpack
#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout -b ch-setup-sass-160011173
```
```sh
$ npm install
$ npm start
```
#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160011173](https://www.pivotaltracker.com/story/show/160011173)

### CODE REVIEW

**Comment by ronaldndirangu** [comment]
The node_modules folder should not be pushed to the remote repo. Please add it to the .gitignore file.

**Review by petermwash — CHANGES_REQUESTED** [review]
Kindly add node_modules in the .gitignore file

**Comment by janetnim** [comment]
In ```How should this be manually tested?``` on your PR, if follow the steps outlined, the app does not run...probably because I need to install dependencies?...maybe add that to your description

File: src/App/index.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
-
+// import './index.scss';
```
Comment (jmnyarega) [inline src/App/index.js]: Remove commented lines

Comment (migot01) [reply src/App/index.js]: cool, its done

**Review by janetnim — APPROVED** [review]

**Review by petermwash — APPROVED** [review]

**Review by dcDalin — APPROVED** [review]

**Comment by jmnyarega** [comment]
Update your branch

**Pull request merged**